### PR TITLE
Redirect home page to beta.webmaker.org & some clean-up

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

File: routes/redirect.js [hunk 1]
```diff
@@ -1,54 +1,23 @@
...
-      '/login'
-    ]
-  }, {
     route: '/tools',
     paths: [
       '/tools/x-ray-goggles',
       '/tools/x-ray-goggles/install'
     ]
   }, {
-    route: '/party',
```
Comment (ScottDowne) [inline routes/redirect.js]: I like to party.

**Comment by ScottDowne** [comment]
R+

**Comment by thisandagain** [comment]
This looks good to go once https://github.com/mozilla/webmaker-desktop/issues/113 lands.
/cc @k88hudson @xmatthewx

**Comment by jbuck** [comment]
@thisandagain this has been landed on staging: https://webmaker.mofostaging.net . Good to push to prod? (beta.webmaker.org lgtm, just want a final sign-off

**Pull request merged**